firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/internal/SignInResultActivity.java

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
```

Comment:
Is it possible to also move InstallActivity to this internal package?

Reply:
Yes it is. I had to copy logwrapper since we want logwrapper package private and install activity uses a good amount of logging.

Reply:
My instinct is to delete the package private logwrapper and exclusively the public one in the internal package. Since we don't document anything in internal, I think the customer will be unlikely to accidentally use the internal package. 

What do you think?